### DISCUSSION TITLE
Works but doesn't work

### DISCUSSION THREAD

**Discussion by 44cK3r** [skyline-emu/skyline#1244] [General]
The app / emulator works but when launching a ROM the emulation either crashes but the app doesn't which is somewhat good or black screen but when you have it show the FPS or stats or whatever you call it I forget sorry it does what it normally does as if it was normally emulating a ROM even Sonic Mania doesn't work anymore either way love you all keep it up and stay positive

**Comment by finr0y** [comment]
known issue, mark is currently asleep, he will probably fix when he returns

**Reply by 44cK3r** [reply]
Ok all good thank you love you

**Reply by finr0y** [reply]
you got it bro thanks for listening
